### ISSUE TITLE
Middleware CacheRequests does not exist 

### ISSUE THREAD

**Issue by basdog22** [barryvdh/laravel-httpcache#19]
Hi Barry!
Just installed with composer:
"barryvdh/laravel-httpcache": "0.2.x@dev"

and the middleware 'Barryvdh\HttpCache\Middleware\CacheRequests' does not exist.

Am i missing something?

**Comment by basdog22** [comment]
Nevermind i set up one by myself.
Thanks for reading :)

**Comment by argb** [comment]
I found the same problem too,does this package still be maintained?

**Comment by argb** [comment]
Maybe not a problem,found this below:

public function boot()
    {
    	/** @var \Barryvdh\StackMiddleware\StackMiddleware $stack */
        $stack = app('Barryvdh\StackMiddleware\StackMiddleware');
        
        $stack->bind(
          'Barryvdh\HttpCache\Middleware\CacheRequests',
          'Symfony\Component\HttpKernel\HttpCache\HttpCache',
          [
            $this->app['http_cache.store'],
            $this->app['http_cache.esi'],
            $this->app['http_cache.options']
          ]
        );
    }

**Comment by barryvdh** [comment]
Did you add the service provider?

**Comment by basdog22** [comment]
Yes. I add SP and also added the middleware to the kernel but the httpcache did not work. No files in the storage dir or age value in headers.

I created a middleware that takes the response with $next($request) and then i added ->setTtl to it. Now it is ok

Info: My dir structure is not the default laravel 5

**Comment by khoas3** [comment]
I got the same issue, how can I fix this?

**Comment by khoas3** [comment]
Never mind, my failure setting up.

**Comment by ctf0** [comment]
this is an old ticket but incase someone was lost like myself
```php
protected $middleware = [
        'Barryvdh\HttpCache\Middleware\CacheRequests',
    ];

protected $routeMiddleware = [
        'ttl' => \Barryvdh\HttpCache\Middleware\SetTtl::class,
    ];
```
and either
```php
Route::get('my-page', function(){
   return 'Hello' 
})->middleware('ttl:60'); // Cache 1 minute
```

or global https://github.com/barryvdh/laravel-httpcache/issues/28#issuecomment-323553368
